brave/brave-core
# Fix custom ad notification z-ordering and possible display issues.

Resolves https://github.com/brave/brave-browser/issues/22203



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

The issue should be tested on Windows and MacOS platforms.

- Clean install
- Run Brave with: `--enable-features="CustomAdNotifications:should_attached_ad_notification_to_browser_window/true"`
- Enable rewards and ads
- Trigger a notification ad
- Check that the notification ad has the same Z-order as the Browser window
